cleanlab/internal/neighbor/neighbor.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
```

Comment:
[nit] name of this file is a bit vague, given it's in the neighbor/ subfolder already. Could consider knn_graph.py or knearestneighbors.py something more explicit like that